src/microns_to_nwb/tools/behavior/behavior.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+from hdmf.backends.hdf5 import H5DataIO
+from phase3 import nda
+from pynwb import TimeSeries
+from pynwb.behavior import PupilTracking, SpatialSeries, EyeTracking
+
+
+def add_eye_tracking(scan_key, nwb):
+    pupil_min_r, pupil_maj_r, pupil_x, pupil_y, timestamps = (nda.RawManualPupil() & scan_key).fetch1(
+        "pupil_min_r", "pupil_maj_r", "pupil_x", "pupil_y", "pupil_times"
+    )
+
+    pupil_min_r = TimeSeries(
+        name="pupil_min_r",
+        description="minor axis of pupil tracking ellipse",
+        data=H5DataIO(pupil_min_r, compression=True),
+        timestamps=H5DataIO(timestamps, compression=True),
+        unit="n.a.",
+    )
+
+    pupil_maj_r = TimeSeries(
+        name="pupil_maj_r",
+        description="jajor axis of pupil tracking ellipse",
+        data=H5DataIO(pupil_maj_r, compression=True),
+        timestamps=pupil_min_r,
+        unit="n.a.",
+    )
+
+    pupil_tracking = PupilTracking([pupil_min_r, pupil_maj_r])
+    nwb.add_acquisition(pupil_tracking)
+
+    pupil_xy = SpatialSeries(
+        name="eye_position",
+        description="x,y position of eye",
+        data=H5DataIO(np.c_[pupil_x, pupil_y], compression=True),
+        timestamps=pupil_min_r,
+        unit="n.a.",
```

Comment:
Still in units pixels apparently? I wish it was known how big a pixel is...

Ahhhh wait this is known apparently; do ctrl+f on `um_width`, its a field somewhere for each session. Not that we should scale things ourselves, but good to know someone could do it if they wanted to. (I'd just make sure that field makes it somewhere in the NWB file)